react/index.web.js

```diff
@@ -51,6 +54,16 @@ window.addEventListener('beforeunload', () => {
         APP.logCollector.stop();
         APP.logCollectorStarted = false;
     }
+    if (APP.conference.roomName) {
+        const state = APP.store.getState();
+
+        // The conference we have already joined or are joining.
+        const conference_ = getCurrentConference(state);
+
+
+        APP.store.dispatch(conferenceWillLeave(conference_));
```

Comment:
I'm wondering isn't there a better place for dispatching this action. The `beforeunload` doesn't seem very related to the conference state to me.